csrankings-r.csv

```diff
@@ -1383,6 +1385,7 @@ Ryan Kok Leong Ko,University of Queensland,https://researchers.uq.edu.au/researc
 Ryan Layer,University of Colorado Boulder,http://layerlab.org,AXnmFTIAAAAJ
 Ryan M. Layer,University of Colorado Boulder,http://layerlab.org,AXnmFTIAAAAJ
 Ryan Marcus,University of Pennsylvania,https://rmarcus.info/,vPOl-IwAAAAJ
+Ryan Newton, Purdue University, https://www.cs.purdue.edu/people/faculty/rrnewton.html, Bp6dm-oAAAAJ

```

Comment:
All spaces need to be removed after commas (in every entry). Per LinkedIn, Ryan is not a TT faculty member at Purdue (listed as Adjunct) - please clarify.

Reply:
Please address this issue ASAP; I am closing old PRs by the end of this week.

Reply:
Ryan Newton is an TT Associate Professor in the Department of Computer Science and in the School of Electrical and Computer Engineering at Purdue University. 
https://www.cs.purdue.edu/people/faculty/rrnewton.html
https://engineering.purdue.edu/ECE/People/ptProfile?resource_id=247461

Spaces have been removed.